src/main.ts

```diff
@@ -12,33 +12,35 @@ loadLocal();
 
 const scriptRegex = /generate-([a-zA-Z\\-]+)\.ts/;
 
-const excludedScripts = ['pretty-manifest'];
+const excludedScripts = ['pretty-manifest', 'font-glyph-enums', 'symbols'];
 
 // These scripts generate data needed by other scripts,
 // so they need to run first in this order
-const prioritizedScripts = [
-  'enums',
-  'season-info',
-  'source-info',
-  'watermark-info',
-  'font-glyph-enums',
-];
+const prioritizedScripts = ['enums', 'season-info', 'source-info', 'watermark-info'];
 // If a script outputs one of these files, compile it
 const toCompileOutputs = ['generated-enums.ts', 'd2-font-glyphs.ts'];
 // These files should be copied verbatim from data/ to output/
-const copyDataToOutput = ['legacy-triumphs.json', 'stat-effects.ts'];
+const copyDataToOutput = ['legacy-triumphs.json', 'stat-effects.ts', 'd2-font-glyphs.ts'];
```

Comment:
Nice idea - but the `copyDataToOutput` loop needs to be moved to after the data generation step then.